ndindex/ndindex.py

```diff
@@ -2,6 +2,9 @@
 
 from numpy import ndarray
 
+newaxis = None
```

Comment:
This needs to be implemented as an NDIndex subclass.

Reply:
OK, I just went with that NumPy does. Let me know if this is what you had in mind